pipeline/inputs/standard-input.md

```diff
@@ -1,18 +1,37 @@
 # Standard Input
 
-The **stdin** plugin allows to retrieve valid JSON text messages over the standard input interface \(stdin\). In order to use it, specify the plugin name as the input, e.g:
+The **stdin** plugin allows to retrieve a message stream from the standard input interface \(stdin\) of the fluent-bit process. In order to use it, specify the plugin name as the input, e.g:
 
 ```bash
 $ fluent-bit -i stdin -o stdout
 ```
 
-As input data the _stdin_ plugin recognize the following JSON data formats:
+If the stdin stream is closed (end-of-file), the stdin plugin will instruct
+`fluent-bit` to exit with success (0) after flushing any pending output.
+
+## Input formats
+
+If no parser is configured for the stdin plugin, it expects *valid json* input input data with one of the following data formats:
```

Comment:
```suggestion
If no parser is configured for the stdin plugin, it expects *valid json* input data with one of the following data formats:
```

I'm not sure those are valid JSON though - the `=>` is something `stdout` uses?

Reply:
@patrick-stephens The output here isn't json because the example uses the default format of the `stdout` plugin. The _input_ to the `stdin` plugin is required to be valid json unless a parser is configured explicitly.